cmd/helm/deployment.go

```diff
@@ -17,18 +17,24 @@ limitations under the License.
 package main
 
 import (
+	"errors"
+	"regexp"
+
 	"github.com/codegangsta/cli"
+	"github.com/kubernetes/helm/pkg/format"
 )
 
 func init() {
+	//addCommands(listCmd())
```

Comment:
If people are okay with not having this alias to `helm list`, I want to remove this stuff. I think it's terribly confusing to have a `helm list` command, given that you can list several different things.